src/test/scala/mesosphere/marathon/core/matcher/base/util/ActorOfferMatcherTest.scala

```diff
@@ -67,16 +72,16 @@ class ActorOfferMatcherTest extends AkkaUnitTest {
   }
 
   class Fixture {
-    def offerWithFaultRegion(faultRegion: String) = {
+    def offerWithFaultRegion(faultDomain: FaultDomain) = {
       Offer.newBuilder()
         .setId(MesosOfferIdProto.newBuilder().setValue(UUID.randomUUID().toString))
         .setFrameworkId(FrameworkID.newBuilder().setValue("notanidframework"))
         .setSlaveId(SlaveID.newBuilder().setValue(s"slave1"))
         .setHostname("hostname")
         .setDomain(DomainInfo.newBuilder()
-          .setFaultDomain(FaultDomain.newBuilder()
-            .setRegion(RegionInfo.newBuilder().setName(faultRegion))
-            .setZone(ZoneInfo.newBuilder().setName("zone")))
+          .setFaultDomain(FaultDomainPB.newBuilder()
```

Comment:
nit: dislike the `PB` in `FaultDomainPB`. May be `Protos.FaultDomain` or similar?

Reply:
+1

Reply:
Well, it will be a bit wordy: `Protos.DomainInfo.FaultDomain`